### PR TITLE
Abasive Tech Customer Story

### PR DESCRIPTION
**Pull request by robmarcer**

### CODE REVIEW

**Review by gdziuba — APPROVED** [review]
Looks great.  Great reference.

**Comment by gdziuba** [comment]
@Hasmin-AC @robmarcer

Do we have a time for when we want to promote?

File: src/customer-stories/leveraging-node-red-and-flowfuse-to-automate-precision-manufacturing.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    results:
+        - Significant impacted on Abrasive Technology's business operations
+        - Delivered projects which previously may have not been practical
+---
+
+In the competitive world of manufacturing, where precision, innovation, and efficiency reign supreme, Abrasive Technology has emerged as a global leader in super-abrasive tooling and coatings for a wide variety of applications. 
+
+<!--more-->
+
+This case study provides an in-depth exploration of how Abrasive Technology harnessed the power and flexibility of Node-RED to optimize automation, streamline communication with machinery, and accelerate development processes.
```
Comment (ZJvandeWeg) [inline src/customer-stories/leveraging-node-red-and-flowfuse-to-automate-precision-manufacturing.md:24]:
```suggestion
This case study provides an in-depth exploration of how Abrasive Technology harnessed the power and flexibility of FlowFuse and Node-RED to optimize automation, streamline communication with machinery, and accelerate development processes.
```

File: src/customer-stories/leveraging-node-red-and-flowfuse-to-automate-precision-manufacturing.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+#### Environmental Considerations
+
+Manufacturing environments, particularly those involving electro-coating processes in plating labs with high humidity and corrosion, presented unique challenges. However, Node-RED's adaptability allowed Abrasive Technology to address these concerns effectively. Dudley emphasizes, "Environmental considerations are crucial. Node-RED facilitates placing our in-house control equipment in sealed boxes to ensure longevity."
+
+#### Hardware Challenges and Solutions
+
+Abrasive Technology initially utilized Raspberry Pis for hosting Node-RED instances, linking them to industrialized touch screens for Human-Machine Interfaces (HMIs). While Raspberry Pis proved reliable for many applications, certain projects pushed hardware capabilities. Dudley highlights, "I have found them to be very reliable, but on the current project, I'm absolutely bumping up against its hardware capabilities." The anticipation of the Pi5, with enhanced capabilities, holds promise, and Dudley is eagerly awaiting its arrival.
```
Comment (ZJvandeWeg) [inline src/customer-stories/leveraging-node-red-and-flowfuse-to-automate-precision-manufacturing.md:46]:
```suggestion
Abrasive Technology initially utilized Raspberry Pis for hosting Node-RED instances, linking them to industrialized touch screens for Human-Machine Interfaces (HMIs). While Raspberry Pis proved reliable for many applications, certain projects pushed hardware capabilities. Dudley highlights: "I have found them to be very reliable, but on the current project, I'm absolutely bumping up against its hardware capabilities." The anticipation of the Pi5, with enhanced capabilities, holds promise, and Dudley is eagerly awaiting its arrival.
```

File: src/customer-stories/leveraging-node-red-and-flowfuse-to-automate-precision-manufacturing.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+In addition to Raspberry Pis, Abrasive Technology currently operates a server running Node-RED locally, with additional instances in locally installed FlowFuse. Dudley notes, “We are now actively transitioning our Node-RED instances from locally maintained servers to the FlowFuse cloud.”
```
Comment (ZJvandeWeg) [inline src/customer-stories/leveraging-node-red-and-flowfuse-to-automate-precision-manufacturing.md:48]:
```suggestion
In addition to Raspberry Pis, Abrasive Technology currently operates a server running Node-RED locally, with additional instances in locally installed FlowFuse. Dudley notes, “We are now actively transitioning our Node-RED instances from locally maintained servers to the FlowFuse Cloud.”
```

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

**Comment by Hasmin-AC** [comment]
In the summary can we add a line or two on how FlowFuse helped too.

**Comment by ZJvandeWeg** [comment]
@robmarcer Yes, last question from my side is if you updated the PDF for download too?

**Review by ZJvandeWeg — APPROVED** [review]

**Comment by robmarcer** [comment]
I've updated the PDF @ZJvandeWeg, thanks for the reminder.

**Pull request merged**